09. cli-and-http-request/commandhelper/builders/LogWriterBuilder.go

```diff
@@ -0,0 +1,29 @@
+package commandhelper
+
+import (
+	. "github.com/nafrose/exploring/clirunner/commandhelper/interfaces"
+	. "github.com/nafrose/exploring/clirunner/commandhelper/structs"
+)
+
+type LogWriterBuilder struct {
+	*WritersCollectionBuilder
+}
+
+func (lwb *LogWriterBuilder) Add(logWriter *LogWriter) *LogWriterBuilder {
+	lwb.OutputLogger().Add((*logWriter).OutputWriter())
```

Comment:
lwb.OutputLogger().Add((*logWriter).**OutputWriter())** << dont understand where it is coming from. cant see the source